src/activities/categorization/Categorization.qml

```diff
@@ -0,0 +1,229 @@
+                            }
+                            items.easyMode = checked
+                        }
+                    }
+                    
+                    GCDialogCheckBox {
+                        id: mediumModeBox
+                        width: 250 * ApplicationInfo.ratio
+                        text: qsTr("Medium mode")
+                        checked: items.mediumMode
+                        onCheckedChanged: {
+                            print("Medium Mode: ",checked)
+                            if(items.easyMode == true){
+                                items.easyMode = false
+                                easyModeBox.checked = false
+                                items.scoreChecked = true
+                                items.instructionsChecked = false
+                                items.categoryImageChecked = false
+                            }
+                            else if(items.expertMode == true){
+                                items.expertMode = false
+                                expertModeBox.checked = false
+                                items.instructionsChecked = false
+                                items.categoryImageChecked = false
+                                items.scoreChecked = true
+                            }
+                            items.mediumMode = checked
+                        }
+                    }
+                    
+                    GCDialogCheckBox {
+                        id: expertModeBox
+                        width: 250 * ApplicationInfo.ratio
+                        text: qsTr("Expert mode")
+                        checked: items.expertMode
+                        onCheckedChanged: {
+                            print("Score Diabled: ",checked)
+                            if(items.easyMode == true){
+                                items.easyMode = false
+                                easyModeBox.checked = false
+                                items.instructionsChecked = false
+                                items.categoryImageChecked = false
+                                items.scoreChecked = false
+                            }
+                            else if(items.mediumMode == true){
+                                items.mediumMode = false
+                                mediumModeBox.checked = false
+                                items.scoreChecked = false
+                            }
+                            items.expertMode = checked
+                        }
+                    }
+                }
+            }  
+            onLoadData: {
+                items.easyMode = dataToSave["easyMode"] === "true" ? true : false
+                items.mediumMode = dataToSave["mediumMode"] === "true" ? true : false
+                items.expertMode = dataToSave["expertMode"] === "true" ? true : false
+                items.scoreChecked = dataToSave["score"] === "true" ? true : false
```

Comment:
I'm not sure if is needed to store score, instructions and images as they are computable from the mode